### PR TITLE
Update azure-portal-safelist-urls.md

### PR DESCRIPTION
**Pull request by segormle**
Add appmanagement.activedirectory.microsoft.com to allow list for Azure portal. Note that the *.msidentity.us and *.msidentity.cn endpoints specified under US Government Cloud and Microsoft Azure operated by 21Vianet Cloud cover the national cloud equivalents for this endpoint, and so no URLs are added to that section.

### CODE REVIEW

File: articles/azure-portal/azure-portal-safelist-urls.md [hunk 1]
```diff
@@ -80,6 +80,7 @@ api.aadrm.com (Microsoft Entra)
 api.loganalytics.io (Log Analytics Service)
 api.azrbac.mspim.azure.com (Microsoft Entra)
 *.applicationinsights.azure.com (Application Insights Service)
+appmanagement.activedirectory.microsoft.com (Microsoft Entra)
```
Comment (DavidParks8) [inline articles/azure-portal/azure-portal-safelist-urls.md:83]: looks like only the public cloud was updated. Wasn't the intention to also add mc and usgov lists?

**Comment by Jak-MS** [comment]
@JnHs 
Can you review the proposed changes? 

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: articles/azure-portal/azure-portal-safelist-urls.md (hunk 1 above)
Comment (segormle) [reply articles/azure-portal/azure-portal-safelist-urls.md:83]: *.msidentity.us and *.msidentity.cn cover the national cloud scenarios. Specifically, appmanagement.msidentity.us and appmanagement.msidentity.cn.

**Comment by segormle** [comment]
[like]  Sean Gormley reacted to your message:
________________________________

**Comment by Jak-MS** [comment]
@JnHs - nudge. can you review? thanks.

**Comment by JnHs** [comment]
> @JnHs - nudge. can you review? thanks.

Waiting for PMs to confirm, will reply back ASAP!

**Comment by segormle** [comment]
[like]  Sean Gormley reacted to your message:
________________________________

**Review by JnHs — APPROVED** [review]

**Comment by JnHs** [comment]
#sign-off

**Pull request merged**